### PR TITLE
B/add 4pool

### PR DESCRIPTION
**Pull request by memacius32**
Adding 4 pool (USDC/USDT/WUST/FRAX) Info and Relevant tokens.

### CODE REVIEW

**Review by rej156 — APPROVED** [review]
// NOTE - Will need external tab hyperlink to recognise if the user has the rainbow Bridge frax token version in their wallet balance 

then display the link to the frax website to convert it over somewhere 
maybe on the stableswap add liquidity page

File: dev/generate_tokens/base_tokens_map.js [hunk 1]
```diff
@@ -95,5 +95,19 @@ module.exports = {
       'AAVE',
       'Aave Token'
     )
+  },
+  // TODO: Temporally adding here, will need to add to tokens repo and then clean this up.
+  WUST: {
+    [ChainId.AURORA]: new Token(
```
[the remark this answers is not in the document]
Comment (memacius32) [reply dev/generate_tokens/base_tokens_map.js:101]: As I undersand it, I think the problem is with pushing commented code, not TODOS (if we need to remind ourselves of things we need to do later), 

Although in this case, i can inded add WUST from the token repo and pull it from there, so I will remove this one,

File: dev/generate_tokens/base_tokens_map.js [hunk 2]
```diff
@@ -95,5 +95,19 @@ module.exports = {
...
+      ChainId.AURORA,
+      '0x8D07bBb478B84f7E940e97C8e9cF7B3645166b03',
+      6,
+      'WUST',
+      'UST (WormHole)'
+    )
+  },
+  // TODO: Temporally adding correct FRAX address here. Will probably need to replace in our tokens repo.
```
[the remark this answers is not in the document]
Comment (memacius32) [reply dev/generate_tokens/base_tokens_map.js:109]: I will keep this one until the $frax token problem is tackled (I don'tk now yet if we need to add the canonical address to our token list or what)

[the remark this answers is not in the document]
Comment (rej156) [reply dev/generate_tokens/base_tokens_map.js:109]: yeah canonical one is the correct one which is this but we need to do the note below so we need both but do not display the rainbow one in our tokenlist for swaps

**Pull request merged**